config/config_as_cty.go

```diff
@@ -68,6 +68,14 @@ func terragruntConfigAsCty(config *TerragruntConfig) (cty.Value, error) {
 		output["generate"] = generateCty
 	}
 
+	retryableCty, err := gostructToCty(config.RetryableErrors)
```

Comment:
This was at first confusing to me because `RetryableErrors` is not a struct, but then I realized that `ImpliedType` "does the right thing." We should probably rename `gostructToCty` eventually...

Reply:
I went through the very same thought process... :)

Reply:
Might want a GitHub issue for this too.

Reply:
Done: https://github.com/gruntwork-io/terragrunt/issues/1384